TOC-tidb-cloud.md

```diff
@@ -46,6 +46,9 @@
     - Ruby
       - [mysql2](/develop/dev-guide-sample-application-ruby-mysql2.md)
       - [Rails](/develop/dev-guide-sample-application-ruby-rails.md)
+    - Serverless JavaScript Driver (Beta)
+      - [TiDB Serverless Driver](/tidb-cloud/serverless-driver.md)
```

Comment:
How about TiDB Cloud Serverless Driver?

By retaining this approach, we can ensure the availability of serverless drivers across all TiDB Cloud product lines, while explicitly specifying that the driver is designed for utilizing TIDB Cloud in a serverless environment.

@zhangyangyu

Reply:
+1